sci-libs/libmed/libmed-4.0.0-r1.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# EAPI=7 uses ninja generator by default but it's incompatible with USE=fortran
+# https://github.com/Kitware/ninja/tree/features-for-fortran#readme
+CMAKE_MAKEFILE_GENERATOR=emake
+
+FORTRAN_NEEDED=fortran
+# NOTE:The build for multiple python versions should be possible but complecated for the build system
+PYTHON_COMPAT=( python2_7 python3_{5,6} )
+
+inherit cmake-utils fortran-2 python-single-r1
+
+MY_P="med-${PV}"
+
+DESCRIPTION="A library to store and exchange meshed data or computation results"
+HOMEPAGE="https://www.salome-platform.org/"
+SRC_URI="http://files.salome-platform.org/Salome/other/${MY_P}.tar.gz"
+
+LICENSE="GPL-3 LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc fortran mpi python test"
+
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+RDEPEND="
+	>=sci-libs/hdf5-1.10.0[fortran=,mpi=]
+	mpi? ( virtual/mpi[fortran=] )
+	python? ( ${PYTHON_DEPS} )
+"
+DEPEND="${RDEPEND}
+	python? ( >=dev-lang/swig-2.0.9:0 )
+"
+
+S="${WORKDIR}"/${MY_P}
+
+#	"${FILESDIR}/${P}-hdf5-1.8-support.patch"
+PATCHES=(
+	"${FILESDIR}/${PN}-3.3.1-cmake-fortran.patch"
+	"${FILESDIR}/${PN}-3.3.1-disable-python-compile.patch"  # managed by function of python eclass
+	"${FILESDIR}/${PN}-3.3.1-mpi.patch"
+	"${FILESDIR}/${P}-cmakelist.patch"
+	"${FILESDIR}/${P}-tests.patch"  # disable a few tests not running
+	"${FILESDIR}/${P}-installdoc.patch"
+)
+
+DOCS=( AUTHORS COPYING COPYING.LESSER ChangeLog NEWS README TODO )
```

Comment:
we don't need the `COPYING*` stuff installed

Reply:
fixed